### PR TITLE
commited file with suggested changes

### PR DESCRIPTION
**Pull request by JyothiKumar03**
removed the brackets content

Closes #

### What new changes did you make? Tick all applicable boxes
- [ X] Added new abbreviation
- [ ] Fixed something in the source code
- [ ] Added a new feature
- [ ] Fixed the docs (README.md, CONTRIBTUING.md, etc)

### Describe the new changes you added.
I removed the extra content in the brackets. Sorry I was busy with the exams. 
Hope this one is good to go!!

### CODE REVIEW

File: public/server/db/gn.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "definition": "Good Night",
+  "alternatives": "Get Naked
```
Comment (Njong392) [inline public/server/db/gn.json:3]: You missed a quote at the end here.
```suggestion
  "alternatives": "Get Naked"
```

**Review by Njong392 — CHANGES_REQUESTED** [review]
Thanks for the new changes. Just one last suggested change and we are good to go. 

You don't need to create a new pull request if a current one needs adjustment. You can update the file as you normally would, commit, and push the changes, and GitHub will update this pull request for you.

**Comment by JyothiKumar03** [comment]
Hey, I updated the file. Check if it requires any changes and merge it

**Review by Njong392 — APPROVED** [review]
This looks awesome now! Thank you so much for the new addition.

**Pull request merged**